### PR TITLE
Added materialUI library

### PR DESCRIPTION
**Pull request by oleksandr-vovk**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -44,4 +46,4 @@
     "karma-jasmine-html-reporter": "~1.7.0",
     "typescript": "~4.6.4"
   }
-}
+}
```
Comment (FeloniukM) [inline frontend/package.json]: Must be white space

File: frontend/src/shared/shared.module.ts [hunk 2]
```diff
@@ -22,6 +24,7 @@ import { LoadingComponent } from './components/loading-component/loading.compone
         ReactiveFormsModule,
         RouterModule,
         LoadingComponent,        
+        MaterialModule,
     ],
 })
 export class SharedModule { }
```
Comment (FeloniukM) [inline frontend/src/shared/shared.module.ts]: same

File: frontend/src/styles.sass [hunk 3]
```diff
@@ -1 +1,2 @@
 /* You can add global styles to this file, and also import other style files */
+@import '~@angular/material/prebuilt-themes/indigo-pink.css'
```
Comment (FeloniukM) [inline frontend/src/styles.sass]: here too

**Review by ismv — DISMISSED** [review]
LGTM

**Review by Setup1337 — APPROVED** [review]

**Review by ismv — APPROVED** [review]

**Pull request merged**